packages/components/src/InputNumber/InputNumber.test.tsx

```diff
@@ -6,7 +6,7 @@ import { InputNumber, InputNumberRef } from ".";
 afterEach(cleanup);
 
 it("renders an input type number", () => {
-  const tree = renderer.create(<InputNumber value="123" />).toJSON();
+  const tree = renderer.create(<InputNumber value={123} />).toJSON();
```

Comment:
This should have always required a number but, did it get more strict in practice as the result of these changes?

Reply:
This has been like this for a long time. I have noticed it before, but never had a reason to fix it.